internal/restic/node.go

```diff
@@ -478,7 +573,33 @@ func (node Node) sameContent(other Node) bool {
 }
 
 func (node Node) sameExtendedAttributes(other Node) bool {
-	if len(node.ExtendedAttributes) != len(other.ExtendedAttributes) {
+	// Converting the array of ExtendedAttributes to an array of Attribute
+	var attributes []Attribute
```

Comment:
If we want to avoid creation of these temp []Attribute arrays we could copy the full method sameExtendedAttributes and create sameGenericAttributes. Maintainability vs performance hit. Which would be preferred here?